packages/block-library/src/columns/style.scss

```diff
@@ -18,7 +18,9 @@
 
```

Comment:
Should we change the shorthand `flex: 1` to just the property we're using from it, `flex-grow: 1`, here?

A little redundant. This is essentially what we currently have:
```
.wp-block-column {
    flex: 1;
       (flex-grow: 1)
       (flex-basis: 0%)

    flex-basis: 100%;
}
```

Reply:
Good call there I'd say. I still always have to lookup what `Flex:1;`actually means.

Reply:
Actually, I don't think we need to set `flex:1;` or `flex-grow:1;` at all.  `flex-basis:100%;` (or `width:100%;`) is all we need.